app/common/services/endpoints/user-endpoint.js

```diff
@@ -2,11 +2,23 @@ module.exports = [
     '$resource',
     '$rootScope',
     'Util',
+    'CacheFactory',
 function (
     $resource,
     $rootScope,
-    Util
+    Util,
+    CacheFactory
 ) {
+    var cache;
+
+    if (!(cache = CacheFactory.get('userCache'))) {
```

Comment:
why are we doing special handling here but nowhere else? Doesn't CacheFactory have a method to automatically create a cache if it doesn't exist?